scripts/generate_conda_file.sh

```diff
@@ -73,6 +91,8 @@ fi
 # $ python -m ipykernel install --user --name my_env_name --display-name "Python (my_env_name)"
 #
 
+name: recommenders
```

Comment:
does this fix the name? we don't want this, since most users will have several environments. In the documentation we give a different name to each environment: reco_bare, reco_gpu, etc

Reply:
fixed.